MVPproject/MVPproject/RecipesModule/Cell/TypeRecipesCollectionViewCell.swift

```diff
@@ -0,0 +1,98 @@
+// TypeRecipesCollectionViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Ячека типа рецепта
+class TypeRecipesCollectionViewCell: UICollectionViewCell {
```

Comment:
final +
Там на макетах экран подписал как Category, мне кажется это будет чуть более точное имя для ячейки, чем тип
```suggestion
final class RecipesCategoryCell: UICollectionViewCell {
```